packages/expo-updates/android/src/main/java/expo/modules/updates/UpdatesDevLauncherController.kt

```diff
@@ -140,6 +141,29 @@ class UpdatesDevLauncherController : UpdatesInterface {
     )
   }
 
+  override fun storedUpdateIdsWithConfiguration(configuration: HashMap<String, Any>?, context: Context?, callback: UpdateCallback?) {
+    val controller = UpdatesController.instance
+    val updatesConfiguration = UpdatesConfiguration(context, configuration)
+    if (updatesConfiguration.updateUrl == null || updatesConfiguration.scopeKey == null) {
+      callback?.onFailure(Exception("Failed to load update: UpdatesConfiguration object must include a valid update URL"))
+      return
+    }
+    if (controller.updatesDirectory == null) {
+      callback?.onFailure(controller.updatesDirectoryException)
+      return
+    }
+    val databaseHolder = controller.databaseHolder
+    val launcher = DatabaseLauncher(
+      updatesConfiguration,
+      controller.updatesDirectory!!,
+      controller.fileDownloader,
+      controller.selectionPolicy
+    )
```

Comment:
```suggestion
    val updatesDirectory = controller.updatesDirectory
    if (updatesDirectory == null) {
      callback?.onFailure(controller.updatesDirectoryException)
      return
    }
    val databaseHolder = controller.databaseHolder
    val launcher = DatabaseLauncher(
      updatesConfiguration,
      updatesDirectory,
      controller.fileDownloader,
      controller.selectionPolicy
    )
```
You can assign `updatesDirectory` to a variable to avoid the `!!` operator.